### PR TITLE
Support hana-os-clustering in Large Instance

### PR DESCRIPTION
**Pull request by YunziZhangAzure**
This pr is to support hana clustering configuration in large instance.

Since SBD is set up when large instance is provisioned, this part is skipped in the Ansible scripts.

### Test Failover
sollabdsm31:~ # crm_mon -r
Online: [ sollabdsm31 sollabdsm32 ]
Full list of resources:
stonith-sbd     (stonith:external/sbd): Started sollabdsm31
 Clone Set: cln_SAPHanaTopology_H31_HDB01 [rsc_SAPHanaTopology_H31_HDB01]
     Started: [ sollabdsm31 sollabdsm32 ]
 Master/Slave Set: msl_SAPHana_H31_HDB01 [rsc_SAPHana_H31_HDB01]
     Masters: [ sollabdsm31 ]
     Slaves: [ sollabdsm32 ]
 Resource Group: g_ip_H31_HDB01
     rsc_ip_H31_HDB01   (ocf::heartbeat:IPaddr2):       Started sollabdsm31
     rsc_nc_H31_HDB01   (ocf::heartbeat:anything):      Started sollabdsm31

sollabdsm31:~ # service pacemaker stop
sollabdsm32:~ # crm_mon -r
Online: [ sollabdsm32 ]
OFFLINE: [ sollabdsm31 ]
Full list of resources:
stonith-sbd     (stonith:external/sbd): Started sollabdsm32
 Clone Set: cln_SAPHanaTopology_H31_HDB01 [rsc_SAPHanaTopology_H31_HDB01]
     Started: [ sollabdsm32 ]
     Stopped: [ sollabdsm31 ]
 Master/Slave Set: msl_SAPHana_H31_HDB01 [rsc_SAPHana_H31_HDB01]
     Masters: [ sollabdsm32 ]
     Stopped: [ sollabdsm31 ]
 Resource Group: g_ip_H31_HDB01
     rsc_ip_H31_HDB01   (ocf::heartbeat:IPaddr2):       Started sollabdsm32
     rsc_nc_H31_HDB01   (ocf::heartbeat:anything):      Started sollabdsm32

sollabdsm31:~ # service pacemaker start
sollabdsm31:~ # su - h31adm
h31adm@sollabdsm31:/usr/sap/H31/HDB01> sapcontrol -nr 01 -function StopWait 600 10
h31adm@sollabdsm31:/usr/sap/H31/HDB01> hdbnsutil -sr_register --remoteHost=sollabdsm32 --remoteInstance=01 --replicationMode=sync --name=SITEA
sollabdsm31:~ # crm resource cleanup msl_SAPHana_H31_HDB01 sollabdsm31
sollabdsm31:~ # crm_mon -r

Online: [ sollabdsm31 sollabdsm32 ]
Full list of resources:
stonith-sbd     (stonith:external/sbd): Started sollabdsm32
 Clone Set: cln_SAPHanaTopology_H31_HDB01 [rsc_SAPHanaTopology_H31_HDB01]
     Started: [ sollabdsm31 sollabdsm32 ]
 Master/Slave Set: msl_SAPHana_H31_HDB01 [rsc_SAPHana_H31_HDB01]
     Masters: [ sollabdsm32 ]
     Slaves: [ sollabdsm31 ]
 Resource Group: g_ip_H31_HDB01
     rsc_ip_H31_HDB01   (ocf::heartbeat:IPaddr2):       Started sollabdsm32
     rsc_nc_H31_HDB01   (ocf::heartbeat:anything):      Started sollabdsm32

### Test SBD
sollabdsm31:~ # ps aux | grep sbd
root      70110  0.0  0.0 124884 12324 ?        SL   15:43   0:00 sbd: inquisito          r
root      70112  0.0  0.0 124940 12324 ?        SL   15:43   0:00 sbd: watcher:           /dev/disk/by-path/ip-10.20.213.31:3260-iscsi-iqn.1992-08.com.netapp:sn.98a704f9d          fb811e9988d00a098d3b5f1:vs.19-lun-0 - slot: 0 - uuid: 618eb834-c151-4cc1-81fe-4c          d402df27eb
root      70113  0.0  0.0 138444 17600 ?        SL   15:43   0:01 sbd: watcher:           Pacemaker
root      70114  0.0  0.0 147372 33016 ?        SL   15:43   0:00 sbd: watcher:           Cluster
root     146476  0.0  0.0  10812  1648 pts/4    S+   16:02   0:00 grep --color=a          uto sbd
sollabdsm31:~ # kill -9 70110
sollabdsm31:~ #
Broadcast message from systemd-journald@sollabdsm31 (Sat 2020-06-20 16:02:28 UTC):

sbd[70112]: /dev/disk/by-path/ip-10.20.213.31:3260-iscsi-iqn.1992-08.com.netapp:sn.98a704f9dfb811e9988d00a098d3b5f1:vs.19-lun-0:    emerg: do_exit: Rebooting system: reboot

### CODE REVIEW

File: deploy/ansible/roles/sap-media-transfer/tasks/main.yml [hunk 1]
```diff
@@ -9,21 +9,24 @@
   stat: path={{ hana_software_loc }}
   register: extract_path_status
 
-- name: transfer installation media to hanadbnodes
-  copy:
-    src: "{{ item }}"
-    dest: "{{ azure_files_mount_path }}/DB/"
-  with_fileglob:
-    - "{{ azure_files_mount_path }}/DB/*.ZIP" 
+- name: Ensure installation media exists and has been extracted
```
Comment (rukawata) [inline deploy/ansible/roles/sap-media-transfer/tasks/main.yml:12]: The changes in sap-media-transfer seems not related to this PR?

Comment (YunziZhangAzure) [reply deploy/ansible/roles/sap-media-transfer/tasks/main.yml:12]: Correct. Not related to this PR. This change can help save time when running the script twice on a same host. I thought adding this part could help in testing it.

Comment (rukawata) [reply deploy/ansible/roles/sap-media-transfer/tasks/main.yml:12]: I see. Next time maybe make this change before as a separate PR :)
